lib/airbrake-ruby/config.rb

```diff
@@ -167,6 +176,15 @@ def logger=(logger)
       @logger = logger || @logger
     end
 
+    # Sets the default method wrapping style. Must be :chain or :prepend.
+    def default_wrapping_style=(style)
```

Comment:
We probably want to test this method (line 182).

Reply:
Added tests.